docs/community/content/contribute/code-conduct.cn.md

```diff
@@ -51,7 +51,8 @@ chapter = true
  - `ArrayList`，`HashMap`等可能产生扩容的集合类型必须指定集合初始大小，避免扩容。
  - 日志与注释一律使用英文。
  - 注释只能包含javadoc，todo和fixme。
- - 公开的类和方法必须有javadoc，其他类和方法以及覆盖自父类的方法无需javadoc。
+ - 公开的类和方法必须有 javadoc，其他类和方法以及覆盖自父类的方法无需javadoc。
```

Comment:
Should keep space before the second `javadoc`

Reply:
I revert this change, hugo will add whitespace between Chinese word and English word.